fastlane/Pluginfile

```diff
@@ -6,5 +6,5 @@ group :screenshots, optional: true do
   gem 'rmagick', '~> 4.1'
 end
 
-gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', tag: '0.9.14'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', ref: 'a53b7c2'
```

Comment:
⚠️ Just a reminder that this PR should NOT be merged until a new release toolkit has been released and this switched back to new tag (as described in the PR description) though

Reply:
I guess we could put in place a Peril check for this, like we do for our pods.

Every time we do this kind of PRs, either the author writes this or a reviewers points it out. 😄 If we automate it, we'll get the benefit of actually blocking the merge.